ExpenseSplit/HomeScreen.swift

```diff
@@ -0,0 +1,162 @@
+//
+//  HomeScreen.swift
+//  ExpenseSplit
+//
+//  Created by Ankita Katoch on 8/9/24.
+//
+
+import Foundation
+import SwiftUI
+
+struct Friend: Identifiable {
+    let id = UUID()
+    let name: String
+    let avatar: String
+    let balance: String
+    let isOwed: Bool
+}
+
+struct HomeScreen: View {
+    @State private var showSettleBill = false
+        @State private var showCalculateTip = false
+    let friends = [
+        Friend(name: "Elephany", avatar: "elephant", balance: "€92.21", isOwed: false),
+        Friend(name: "Girrafy", avatar: "Girraf", balance: "£20.00", isOwed: true),
+        Friend(name: "Simba", avatar: "Simba", balance: "settled up", isOwed: false),
+        Friend(name: "Jerry", avatar: "jerry", balance: "$17.51", isOwed: false),
+        Friend(name: "Timothy", avatar: "tim", balance: "$87.28", isOwed: true),
+        Friend(name: "Pumba", avatar: "pumba", balance: "₹1,550.50", isOwed: true)
+    ]
+    
+    
+    var body: some View {
+        NavigationView {
+            VStack {
+                HStack {
+                    Spacer()
+//                    Button(action: {
+//                        // Handle add friends action
+//                    }) {
+//                        Text("Add friends")
+//                    }
+//                    .padding()
+                }
+//
+                .navigationBarTitle("Home Screen", displayMode: .inline)
```

Comment:
I noticed there are two navigationTitle modifiers in this view. Since only one is needed to set the title, you might want to remove the extra one to avoid any potential conflicts and keep the code tidy. Just a small suggestion to help keep things streamlined. Keep up the excellent work!
```suggestion
```